testdata/project-v2/config/rbac/admiral_viewer_role.yaml

```diff
@@ -0,0 +1,27 @@
+# permissions to do edit Admiral.
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  creationTimestamp: null
+  name: admiral-editor-role
```

Comment:
Content in all of the viewer role files seem to be duplicates of editor roles.
Please ensure generate_golden.sh has been run.